src/dojos/cd-find-dojo.vue

```diff
@@ -1,24 +1,39 @@
 <template>
   <div class="cd-find-dojo">
-    <div class="cd-find-dojo__panel">
-      <form class="cd-find-dojo__panel-form" @submit.prevent="$router.push({ query: { q: searchCriteria } });">
-        <h1 class="cd-find-dojo__panel-form-header">{{ $t('Find a Dojo to attend') }}</h1>
-        <p class="cd-find-dojo__panel-form-info">{{ $t('Learn technology in an informal, creative and social environment. Find a dojo near you.') }}</p>
+    <div v-bind:class="searchExecuted ? 'cd-find-dojo__panel--reduced' : 'cd-find-dojo__panel'">
```

Comment:
So, I've just realised this is the same across the board... I know this comment may be confusing because it'll probably show up first, but basically, see the other comments about `--reduced` and apply across everything. 😅